docker/dockerfiles/allin1/allin1-ubuntu.Dockerfile

```diff
@@ -39,6 +39,7 @@ RUN apt-get install -yq tzdata && \
 
 # Install perf tool for low-level performance debug
 RUN apt-get install -yq linux-tools-common linux-tools-generic
+RUN echo "export PATH=/usr/lib/linux-tools/5.15.0-60-generic:$PATH" >> /root/.bashrc
```

Comment:
suggest put this in a separate file and put into `/etc/profile.d/` directory.

Reply:
also looks like a few tools in linux-tools-generic are already available in linux-tools-common, what's the specific reason to install both?

Reply:
did you meant to suggest adding this line to `/etc/profile` so every user account login and bash can have this path?

Reply:
yes , but not in /etc/profile, /etc/profile will source all files in `/etc/profile.d/`, you can find following codes in default /etc/profile
```
if [ -d /etc/profile.d ]; then
  for i in /etc/profile.d/*.sh; do
    if [ -r $i ]; then
      . $i
    fi
  done
  unset i
fi
```

Reply:
> also looks like a few tools in linux-tools-generic are already available in linux-tools-common, what's the specific reason to install both?

I need to use `perf` tool. it comes with the `linux-tools-generic`
This is the instruction I followed:
https://askubuntu.com/questions/50145/how-to-install-perf-monitoring-tool

Reply:
ok. /usr/bin/perf from linux-tools-common is just a script to wrap to the right location of linux-tools-generic. lol

Reply:
`etc/profile.d` doesn't work as the scripts there are not executed when running a non-login shell on the container.
 After some research, I found /etc/bash.bashrc are the right places for setting non-login shell env globally.

See the discussions here in you are interested in learning more details:
 https://askubuntu.com/questions/815066/whats-the-difference-between-bashrc-and-etc-bash-bashrc
 https://stackoverflow.com/questions/57320549/do-dockerfile-run-commands-run-in-a-login-shell-environment

Reply:
ok